### ISSUE TITLE
add ControlNet support

### ISSUE THREAD

**Issue by geroldmeisinger** [comfyanonymous/ComfyUI_TensorRT#10]
> ComfyUI TensorRT engines are not yet compatible with ControlNets or LoRAs. Compatibility will be enabled in a future update.

I got a 2x speed boost on SD1. great work! Go! Go! Go!

(issue created just so I can add an alert :)

**Comment by aikitoria** [comment]
Yes, this would be great! Currently I can use TensorRT for the first stage of my workflow (SDXL) but not the second (1.5 + 2x ControlNet) :(

**Comment by astralmedia** [comment]
also would love this!

**Comment by kadirnar** [comment]
+1

**Comment by ConorSAI** [comment]
Also adding my vote for controlnet support
